_api-reference/index-apis/force-merge.md

```diff
@@ -0,0 +1,123 @@
+| `allow_no_indices` | Boolean | If `false`, the request returns an error if any wildcard expression or index alias targets any closed or missing indexes. Default is `true`. |
+| `expand_wildcards` | Specifies the type of indexes to which wildcard expressions can expand. Supports comma-separated values. Valid values are: <br> - `all`: Expand to all open and closed indexes, including hidden indexes. <br> - `open`: Expand to open indexes. <br> - `closed`: Expand to closed indexes. <br> - `hidden`: Include hidden indexes when expanding. Must be combined with `open`, `closed`, or both. <br> - `none`: Do not accept wildcard expressions. <br> Default is `open`. |
+| `flush` | Boolean | Specifies to perform a flush on the indexes after the force merge. A flush ensures the files are persisted to disk. Default is `true`. |
+| `ignore_unavailable` | Boolean | If `true`, OpenSearch ignores missing or closed indexes. If `false`, OpenSearch returns an error if the force merge operation encounters missing or closed indexes. Default is `false`. |
+| `max_num_segments` | Integer | The number of larger segments into which smaller segments are merged. Set this parameter to `1` to merge all segments into one segment. Default behavior performs the merge as necessary. |
+| `only_expunge_deletes` | Boolean | If `true`, the merge operation only expunges segments containing a certain percentage of deleted documents. The percentage is 10% by default and is configurable in the `index.merge.policy.expunge_deletes_allowed` setting. Using `only_expunge_deletes` may produce segments larger than `index.merge.policy.max_merged_segment`, and those large segments may not participate in future merges. For more information, see [Deleted documents](#deleted-documents). Default is `false`. |
+
+#### Example request: Force merge a specific index
+
+```json
+POST /testindex1/_forcemerge
+```
+{% include copy-curl.html %}
+
+#### Example request: Force merge multiple indexes
+
+```json
+POST /testindex1,testindex2/_forcemerge
+```
+{% include copy-curl.html %}
+
+#### Example request: Force merge all indexes
+
+```json
+POST /_forcemerge
+```
+{% include copy-curl.html %}
+
+#### Example request: Force merge a data stream's backing indexes into one segment
+
+```json
+POST /.testindex-logs/_forcemerge?max_num_segments=1
+```
+{% include copy-curl.html %}
+
+#### Example response
+
+```json
+{
+  "_shards": {
+    "total": 2,
+    "successful": 1,
+    "failed": 0
+  }
+}
+```
+
+## Response fields
+
+The following table lists all response fields.
+
+| Field | Data type | Description |
+| :--- | :--- | :--- |
+| `shards` | Object | Contains information about the shards on which the request was executed. |
+| `shards.total` | Integer | The number of shards on which the operation was executed. |
+| `shards.successful` | Integer | The number of shards for which the operation was successful. |
+| `shards.failed` | Integer | The number of shards for which the operation has failed. |
```

Comment:
Would "on which" be better here and on row 122 in order to maintain consistency with the first two rows?